src/node/RealmNode.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,184 @@
+// !$*UTF8*$!
```

Comment:
Can you move the `RealmNode` target to this project since it's really nice to see syntax issues as you work on these files. The target here just runs a shell script.